eval/eval_test.go

```diff
@@ -218,6 +218,7 @@ func TestEval(t *testing.T) {
 		CheckDiags  syntax.Diagnostics `json:"checkDiags,omitempty"`
 		EvalDiags   syntax.Diagnostics `json:"evalDiags,omitempty"`
 		Environment *esc.Environment   `json:"environment,omitempty"`
+		Preview     map[string]any     `json:"preview,omitempty"`
```

Comment:
```suggestion
		CheckJSON     map[string]any     `json:"checkJson,omitempty"`
```

Reply:
(it would also be nice to have `EvalJSON` as a parallel, but we can do that as a follow-up)

Reply:
fixed and added EvalJSON too